supernode/services/senseregister/task.go

```diff
@@ -537,18 +537,19 @@ func (task *Task) ValidateAndRegister(ctx context.Context, ticket []byte, creato
 						return nil
 					}
```

Comment:
should returns here to reply nftRegTxid back to WN , WN is waiting it to do activate

Reply:
Done.